src/code42cli/cmds/devices.py

```diff
@@ -86,6 +87,29 @@ def _deactivate_device(sdk, device_guid, change_device_name, purge_date):
         )
 
 
+def _change_device_activation(sdk, device_guid, cmd_str):
+    _verify_guid_type(device_guid)
+    try:
+        device = sdk.devices.get_by_guid(device_guid)
+        device_id = device.data["computerId"]
+        if cmd_str == "reactivate":
+            sdk.devices.reactivate(device_id)
+        elif cmd_str == "deactivate":
+            sdk.devices.deactivate(device_id)
+        return device
+    except exceptions.Py42NotFoundError:
+        raise Code42CLIError("The device {} was not found.".format(device_guid))
+    except exceptions.Py42ForbiddenError:
+        raise Code42CLIError("Unable to {} {}.".format(cmd_str, device_guid))
+
+
+def _verify_guid_type(device_guid):
```

Comment:
we could actually just make this the callback on the `device_guid_option` and it'll automatically apply wherever we re-use the option.

Reply:
```
def _verify_guid_type(ctx, param, arg):
    if arg is None:
        return
    try:
        int(arg)
    except ValueError:
         raise Code42CLIError("Not a valid guid.")
```